CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Change Log
 
+## [0.23.0](https://github.com/redhat-developer/vscode-xml/milestone/31?closed=1) (December 15, 2022)
+
+### Enhancements
+
+ * Codelens, completion, definition, documentLink, highlighting, references, rename support in RelaxNG file. See [#834](https://github.com/redhat-developer/vscode-xml/pull/834).
+ * Add MEI, TEI, SVG as keywords. See [#811](https://github.com/redhat-developer/vscode-xml/pull/811).
```

Comment:
I would think this should go more under bug fixes, simply because it mainly just affects how the extension is searched on the marketplace. (ie. a metadata improvement).